File: JavaScript/Lv.0_수열과 구간 쿼리 3.js
// JavaScript

function solution(arr, queries) {
    queries.map(([i,j]) => {
        let temp = arr[i];
        [...arr, arr[i] = arr[j]];
        [...arr, arr[j] = temp];
    
    })
    console.log(arr);
    return arr;
}

File: Java/Lv.0_수열과 구간 쿼리 3.java
// Java

class Solution {
    public int[] solution(int[] arr, int[][] queries) {
        int[] answer = arr;
        for (int[] i : queries) {
          int temp = arr[i[0]];
            answer[i[0]] = arr[i[1]];
            answer[i[1]] = temp;

        }
        return answer;
    }
}
